org.mixedrealitytoolkit.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -31,7 +31,7 @@ public class GazePinchInteractor :
         /// <summary>
         /// Holds a reference to the <see cref="TrackedPoseDriver"/> associated to this interactor if it exists.
         /// </summary>
-        private TrackedPoseDriver TrackedPoseDriver => trackedPoseDriver;
+        public TrackedPoseDriver TrackedPoseDriver => trackedPoseDriver;
```

Comment:
I would like to avoid making this public just for testing purposes.  PL=lease let's find a different solution.

Reply:
You are right, changed to protected internal in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/767/commits/b5240732e09ce6a642878dfb8ef304c92d61a498 , resolving.